metadata.rb

```diff
@@ -1334,7 +1334,57 @@
           :description =>  "Sha-1 checksum for the elasticsearch .tar.gz file",
           :type => 'string'
 
+attribute "elastic/opendistro_security/admin/username",
```

Comment:
There is no need to redefine all these attributes here - Since version 0.4 Karamel also checks the `metadata.rb` of the dependent cookbooks

Reply:
Ok, good to know. I just added it here since i found that we are doing this for other cookbooks.